visualization/mlchartlib/src/INumericRange.ts

```diff
@@ -3,6 +3,7 @@ import { RangeTypes } from "./RangeTypes";
 export interface INumericRange {
     // if the feature is numeric
     min: number;
+    avg:number;
```

Comment:
What is the case this is supporting? At the time, I don't think we want to add this here, as average is a cohort-level metric, not a metric about the range of values the column can take

Reply:
I see. I don't think it makes sense necessarily since average is population based, and a cohort is a sub population. Lets not add this in here at this time

Reply:
The idea of the INumericRange object is to specify the type (numeric, integer) and the min and max ranges encountered in the dataset for bounds checking when adding filters and binning

Reply:
Removing it from the UI and this file.